modules/end-user-guide/partials/proc_using-a-git-provider-access-token.adoc

```diff
@@ -0,0 +1,70 @@
+:parent-context-of-using-a-git-provider-access-token: {context}
+
+[id="using-a-git-provider-access-token_{context}"]
+= Using a Git provider access token
+
+If the administrator of your organization's {prod-short} instance has not configured xref:administration-guide:oauth-for-github-gitlab-or-bitbucket.adoc[OAuth for GitHub, GitLab, or Bitbucket], you can apply your personal access token as a {kubernetes} Secret.
+
+Mounting your access token as a Secret enables the {prod-short} Server to access the remote repository that is cloned during workspace creation, including access to the repository's `/.che` and `/.vscode` folders.
+
+Apply the {kubernetes} Secret in your user {orch-namespace} of the {orch-name} cluster of your organization's {prod-short} instance.
+
+After you have applied the Secret, you can create new workspaces from a private GitHub, GitLab, or Bitbucket-server repository.
+
+TIP: In your user {orch-namespace}, you can create and apply multiple access-token Secrets per a Git provider.
+
+.Prerequisites
+
+* An active `{orch-cli}` session, with administrative permissions, to the {orch-name} cluster. See {orch-cli-link}.
+
+* The link:https://www.gnu.org/software/coreutils/base64[`base64`] command line tools are installed in the operating system you are using.
+
+.Procedure
+
+. Copy your access token and encode it to Base64.
++
+[source,terminal,subs="+quotes,+attributes"]
```

Comment:
```suggestion
[subs="+quotes,+attributes"]
```

Reply:
I've found several places where [source,terminal,  is used, so I think we should merge this as it is and consider to rework all the findings in another PR.